### PR TITLE
Milestone 2: Refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by BenMKT**
Implemented the following:

- Used JavaScript classes instead of objects and arrays.
- Used objects and arrays inside classes.
- Created class methods to add and remove books.
- Added CSS styles to the application to make it match the wireframe.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,51 +1,53 @@
...
-function bookHTML() {
-  const bookSection = document.querySelector('#library');
-  bookSection.innerHTML = '';
-  bookKLibrary.forEach((n) => {
-    bookSection.innerHTML += `<div>
-        <p>${n.title}</p>
-        <p>${n.author}</p>
-        <button type='button' onclick=remove('${n.title}','${n.author}') >Remove</button>
-        <hr>
-    </div>`;
```
Comment (Efiamotu-1) [inline app.js:17]: - [x] From the images below you can see that you are missing the `by` word used in the wireframe please work on correcting your project design :+1:
- wireframe
mb
- your design
yd

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @BenMKT  & @pulgosodog  & @SnakyMz , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (SnakyMz) [reply app.js:17]: Thanks for the review. We will work on the changes as suggested.

Comment (BenMKT) [reply app.js:17]: -Added a `<p>` tag in the `<div class='bookDetail">` to resolve bug

**Review by alexander16108 — APPROVED** [review]
Hi @BenMKT  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
✔️ No linter Errors
✔️ Descriptive README
✔️ Descriptive PR title and description
The code is top-notch! 💯 and has a huge potential.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**